x-pack/plugins/alerting/server/lib/alerting_event_logger/alerting_event_logger.ts

```diff
@@ -45,6 +45,7 @@ interface DoneOpts {
 interface AlertOpts {
   action: string;
   id: string;
+  uuid?: string;
```

Comment:
Same as earlier, I'm wondering in which case `uuid` would not be available. It might be `string` or `undefined` though.

Reply:
Changing this one to remove the optional, since the uuid should always be a string when this is used.